src/web/arr/trove/chart.arr

```diff
@@ -1039,6 +1039,31 @@ default-bar-chart-series = {
   default-interval-color: none
 }
 
+type IntervalChartSeries = {
+  tab :: TableIntern,
+  axisdata :: Option<AxisData>,
+  color :: Option<I.Color>,
+  colors :: Option<RawArray<I.Color>>,
```

Comment:
What's the plural `colors` for here? I don't think it's used in the JS implementation

Reply:
In case we use multiple residuals?

Reply:
Multiple residuals should be handled via multiple instances of `interval-chart`, composed via `render-charts(...)`